src/py-opentimelineio/opentimelineio/console/otiotool.py

```diff
@@ -133,6 +136,19 @@ def main():
 
     # Final Phase: Output
 
+    if args.list_versions:
```

Comment:
Should this happen in phase 1 and return once the list is printed? I didn't use the tool yet, but from the outside it seems weird that printing the versions happens after processing things and that it continues after printing.